bin/ramalama

```diff
@@ -4,6 +4,9 @@ import sys
 
 
 def main():
+    if not sys.stdout.isatty():
+        sys.stdout.reconfigure(line_buffering=True)
```

Comment:
LGTM
I expect stdout buffering was also an issue when running under systemd so not just required for the bats test assertions